### ISSUE TITLE
Make it work cross platform

### ISSUE THREAD

**Issue by jcansdale** [jcansdale/GitPullRequest#11]
There are currently some issues.

bash for Windows

```
jcansdale@DESKTOP-A4A4NCF:~/source/github.com/jcansdale/GitStashFile$ git pr --list

Unhandled Exception: System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> LibGit2Sharp.LibGit2SharpException: this remote has never connected
   at LibGit2Sharp.Core.Ensure.HandleError(Int32 result)
   at LibGit2Sharp.Core.Proxy.git_remote_ls(Repository repository, RemoteHandle remote)
   at LibGit2Sharp.Network.ListReferencesInternal(String url, CredentialsHandler credentialsProvider)
   at GitPullRequest.Services.GitPullRequestService.GetReferences(IRepository repo, String remoteName) in C:\Source\Temp\2018-11-26\GitPullRequest\GitPullRequest.Services\GitPullRequestService.cs:line 139
   at GitPullRequest.Services.GitPullRequestService.GetGitHubRepository(IRepository repo, String remoteName) in C:\Source\Temp\2018-11-26\GitPullRequest\GitPullRequest.Services\GitPullRequestService.cs:line 26
   at GitPullRequest.Services.GitPullRequestService.GetGitHubRepositories(IRepository repo) in C:\Source\Temp\2018-11-26\GitPullRequest\GitPullRequest.Services\GitPullRequestService.cs:line 18
   at GitPullRequest.Program.ListBranches(GitPullRequestService service, Repository repo) in C:\Source\Temp\2018-11-26\GitPullRequest\GitPullRequest\Program.cs:line 89
   at GitPullRequest.Program.OnExecute() in C:\Source\Temp\2018-11-26\GitPullRequest\GitPullRequest\Program.cs:line 44
   --- End of inner exception stack trace ---
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Object[] arguments, Signature sig, Boolean constructor, Boolean wrapExceptions)
   at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   at McMaster.Extensions.CommandLineUtils.Conventions.ExecuteMethodConvention.Invoke(MethodInfo method, Object instance, Object[] arguments) in C:\projects\commandlineutils\src\CommandLineUtils\Conventions\ExecuteMethodConvention.cs:line 86
   at McMaster.Extensions.CommandLineUtils.Conventions.ExecuteMethodConvention.OnExecute(ConventionContext context) in C:\projects\commandlineutils\src\CommandLineUtils\Conventions\ExecuteMethodConvention.cs:line 64
   at McMaster.Extensions.CommandLineUtils.Conventions.ExecuteMethodConvention.<>c__DisplayClass0_0.<<Apply>b__0>d.MoveNext() in C:\projects\commandlineutils\src\CommandLineUtils\Conventions\ExecuteMethodConvention.cs:line 25
--- End of stack trace from previous location where exception was thrown ---
   at McMaster.Extensions.CommandLineUtils.CommandLineApplication.<>c__DisplayClass131_0.<OnExecute>b__0() in C:\projects\commandlineutils\src\CommandLineUtils\CommandLineApplication.cs:line 600
   at McMaster.Extensions.CommandLineUtils.CommandLineApplication.Execute[TApp](CommandLineContext context) in C:\projects\commandlineutils\src\CommandLineUtils\CommandLineApplication.Execute.cs:line 31
   at McMaster.Extensions.CommandLineUtils.CommandLineApplication.Execute[TApp](IConsole console, String[] args) in C:\projects\commandlineutils\src\CommandLineUtils\CommandLineApplication.Execute.cs:line 105
   at GitPullRequest.Program.Main(String[] args) in C:\Source\Temp\2018-11-26\GitPullRequest\GitPullRequest\Program.cs:line 16
error: git-pr died of signal 6
jcansdale@DESKTOP-A4A4NCF:~/source/github.com/jcansdale/GitStashFile$
```

When attempting to browse a pull request:

```
jcansdale@DESKTOP-A4A4NCF:~/source/github.com/jcansdale/GSF$ git pr

Unhandled Exception: System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.ComponentModel.Win32Exception: No such file or directory
   at System.Diagnostics.Process.StartCore(ProcessStartInfo startInfo)
   at System.Diagnostics.Process.Start()
   at System.Diagnostics.Process.Start(ProcessStartInfo startInfo)
   at GitPullRequest.Program.Browse(String pullUrl) in C:\Source\Temp\2018-11-26\GitPullRequest\GitPullRequest\Program.cs:line 127
   at GitPullRequest.Program.BrowsePullRequest(GitPullRequestService service, Repository repo) in C:\Source\Temp\2018-11-26\GitPullRequest\GitPullRequest\Program.cs:line 80
   at GitPullRequest.Program.OnExecute() in C:\Source\Temp\2018-11-26\GitPullRequest\GitPullRequest\Program.cs:line 48
   --- End of inner exception stack trace ---
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Object[] arguments, Signature sig, Boolean constructor, Boolean wrapExceptions)
   at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   at McMaster.Extensions.CommandLineUtils.Conventions.ExecuteMethodConvention.Invoke(MethodInfo method, Object instance, Object[] arguments) in C:\projects\commandlineutils\src\CommandLineUtils\Conventions\ExecuteMethodConvention.cs:line 86
```

System.ComponentModel.Win32Exception

https://twitter.com/bradwilson/status/1069652543150612481

Unknown OS.

![image](https://user-images.githubusercontent.com/11719160/49393426-19995880-f729-11e8-9edc-cc35655a68f8.png)

@bradwilson what were you running ☝️ on?

**Comment by bradwilson** [comment]
Pop!_OS 18.10 (based on Ubuntu 18.10).

**Comment by bradwilson** [comment]
```
 sh$ uname -a
Linux brwilso-linux 4.18.0-11-generic #12-Ubuntu SMP Tue Oct 23 19:22:37 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by jcansdale** [comment]
```
LibGit2Sharp.LibGit2SharpException: this remote has never connected
```
Fixed in #17 

```
System.ComponentModel.Win32Exception: No such file or directory
```
Fixed in #15
